README.md

```diff
@@ -62,6 +62,8 @@ redcap_info:
   enrolment_event: #Unique identifier for the enrolment event
   integration_form: #Name of integration REDCap form
   token: #REDCap API Token used to identify the REDCap user against the REDCap instance
+  attributes:
+    - field_name: #Field name of the attribute specified in REDCap. Multiple field_names are supported.
```

Comment:
is there anywhere the set of field_names are defined? Might be useful to know

Reply:
I think these are very dynamic and can be anything. Also these are defined on the project level. But, yes I agree, once we add support in the aRMT app, we can define here what names are supported by aRMT.